quantum/breast_cancer.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from sklearn import datasets
+from sklearn.decomposition import PCA
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler, StandardScaler
+
+
+def breast_cancer(training_size, test_size, n, plot_data=False):
+    """returns breast cancer dataset
+
+    Wikipedia reference: https://en.m.wikipedia.org/wiki/Breast_cancer
+
+    >>> breast_cancer(10, 4, 7)
+    24.9
```

Comment:
The function return four values: `return sample_train, training_input, test_input, class_labels`